modules/net-lb-proxy-int/README.md

```diff
@@ -0,0 +1,308 @@
+# Internal Application Load Balancer Module
```

Comment:
Miren, can you link the module from root and module-level READMEs?

Reply:
Done